README.md

```diff
@@ -37,26 +50,98 @@ You do not need a RFC for:
+- **Cleaning phase:**
+  - Clean up the RFC document - it should be readable and coherent for a third-party which was not part of the discussions.
+  - Write up the summary focusing on explaining a third-party about the gist of the RFC.
+- **Reviewing phase:**
+  - Ask for input from the non-designated architects and other stakeholders.
+
+You can jump around between phases.
+
+**Disagreement resolution**
+
+> Fight for what you believe, but gracefully accept defeat.
+
+We expect everyone (no matter their role and experience level) to adhere to our [Code of Conduct](https://inveniosoftware.org/governance/) and be open, considerate and respectful during discussions. Search for consensus and be ready to change your mind!
+
+Please do your outmost to not have unresolvable disagreements! The more senior your are, the more responsible you are to not have unresolvable disagreements.
+
+In case all attempts to reach consensus have failed, and really only as a very very last resort, the architects can resolve the conflict by taking a decision. This decision should be properly documented, and can be escalated according to the [Invenio Governance](https://inveniosoftware.org/governance/).
+
+### Step 3: Review the RFC
+
+#### How to submit a pull request with a RFC?
+
+This is used in the step 3 (see "Process overview") to submit the already written RFC.
 
 - Fork this repository.
-- Copy ``0000-template.md`` to ``<product>/0000-<my-title>.md``.
+- Copy ``0000-template.md`` to ``rfcs/<product>-0000-<my-title>.md``.
 - Fill in the RFC.
 - Submit a pull request
 - Update your pull request:
-    - Rename your ``<product>/0000-<my-title>.md`` to ``<product>/<pull request id with leading zeros>-<my-title>.md`` (e.g. ``rdm/0012-persistent-identifiers.md``).
-    - Add a link to the pull request inside the pull request.
+    - Rename your ``<product>-0000-<my-title>.md`` to ``<product>-<pull request id with leading zeros>-<my-title>.md`` (e.g. ``rdm-0012-persistent-identifiers.md``).
+    - Add a link to the pull request inside the RFC.
+
+#### Reviewing checklist
 
-### Process (WIP)
+- Comment on quality of the RFC, not the chosen solutions (this was already done in step 2)!
+- Can the RFC be understood by an experienced third-party that didn't participate in the discussions?
+- Is the RFC coherent?
+- Is the unresolved questions properly documented?
+- Is there sufficient resources to implement the RFC?
 
-The decision process on RFCs have not yet been decided. The plan is to gain
-practical experience from the Invenio RDM project and decide on a process by
-latest July 2020.
+### Step 4: Merge the RFC
 
-Ideas is very welcome (write them on
-https://github.com/inveniosoftware/rfcs/issues/1).
+As soon as the RFC has reached sufficient quality level and consensus it can be merged into this RFC repository. The RFC does not need to be fully completed to be merged, as long as unresolved questions have been listed in the RFC.
```

Comment:
Document RFC states:

- Draft: has reached sufficient quality to be merged, some discussions has happened, but there's open questions 
and it's not ready yet to be implemented.
- Ready: design ready, enough discussions has happened and quality of rfc reached.
- Implemented: the rfc has been implemented in community or code